docs/spec/api.md

```diff
@@ -1,3 +1,7 @@
+page_title: Docker Registry HTTP API V2
```

Comment:
Yep, that's exactly what I wanted. Like I said, it will work on my new stuff too. Thanks Stephen!